buildSrc/src/main/java/Dependencies.kt

```diff
@@ -34,6 +34,8 @@ object Versions {
 
     const val mozilla_appservices = "0.15.0"
     const val servo = "0.0.1.20181017.aa95911"
+
+    const val work_testing = "1.0.0-beta05"
```

Comment:
I assume that this should always have the same version as the workmanager dependency - so we may be able to reuse `Versions.workmanager`.